app/src/main/java/com/github/se/polyfit/data/local/dao/MealDao.kt

```diff
@@ -37,6 +38,9 @@ interface MealDao {
     return meal?.toMeal()
   }
 
+  @Query("SELECT * FROM MEALTABLE WHERE createdAt >= :date ")
```

Comment:
I think we will also need a method to get meals created on a specific date for the daily recap screens, as users will be able to select yesterday for example and see what they ate only that day